app/src/main/java/org/eyeseetea/malariacare/domain/entity/Header.java

```diff
@@ -10,23 +12,14 @@
 
     public Header(long id, String shortName, String name,
             Form form, int index) {
-        this.id = id;
-        this.shortName = shortName;
-        this.name = name;
-        this.form = form;
-        this.index = index;
-    }
 
-    public Header() {
+        setId(required(id,"id is required"));
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class